http.go

```diff
@@ -40,10 +40,14 @@ type CallSummary struct {
 	HTTPResponseBody string
 	// Keep a record of how many http requests were attempted
 	Attempts int
+	// Keep a copy of the URL that was used
+	URL string
+	// Keep a copy of the HTTP Method used
+	Method string
```

Comment:
These two pieces of data are already in `HTTPRequest` member of this struct, so we don't need to add them.

Reply:
I didn't notice!  good call.